src/isar/apis/models/start_mission_definition.py

```diff
@@ -25,6 +26,7 @@ class InspectionTypes(str, Enum):
 
 
 class StartMissionInspectionDefinition(BaseModel):
+    id: Optional[str]
```

Comment:
I think `Optional` parameters should be after all the mandatory parameters

Reply:
fixed